README.md

```diff
@@ -128,6 +128,7 @@ CHANGELOG
 0.5.0
 
 - gh-63: The `roots` method of `arb_poly`, and `nmod_poly` is no longer supported. Use `acb_roots(p).roots()` to get the old behaviour of returning the roots as `acb`. Note that the `roots` method of `fmpz_poly` and `fmpq_poly` currently returns the complex roots of the polynomial.
+- gh-72: The roots method of `arb_poly` is not supported. Use either the `complex_roots` method or `acb_roots(p).roots()` to get the old behaviour of returning the complex roots. The `roots` method on `fmpz_poly` and `fmpq_poly` now return integer and rational roots respectively. To access complex roots on these types, use the `complex_roots` method. For `acb_poly`, both `roots` and `complex_roots` behave the same.
```

Comment:
We should also add notes for other PRs here.

- #83
- #80
- #78 #79

Big refactor across several versions:

- #75 #77 #67 #61

For 0.4.4 there should be

- #71 (adds Cython files again in sdist

For 0.4.3

- #69 (adds OSX arm64 wheels)

We should get better at adding release notes here. It is important for people to see these so that they can see that development is happening. Also I think that each note should list the names of the authors. It is informative for anyone who wants to understand the project to see that there are multiple people working on things.

Reply:
#63 was for 0.4.3 not 0.5.0

For 0.4.2

- #57 (Adds manylinux wheels)

For 0.4.1

- #48 Removes Linux wheels, updates instructions for building from source.

For 0.4.0

- #45 Adds wheels for Windows, OSX and manylinux but the Linux wheels are broken.

Reply:
For 0.4.3 #74 should be noted.